### DISCUSSION TITLE
Support `withConverter` in Firestore API

### DISCUSSION THREAD

**Discussion by aloukissas** [invertase/react-native-firebase#5906] [Ideas]
This already exists in the web and nodejs SDKs (e.g. see [here](https://googleapis.dev/nodejs/firestore/latest/Query.html#withConverter)). Would be amazing if this was available in RNFirebase!

**Comment by mikehardy** [comment]
There is a start on this that is available for continuation + fixup and we could merge. It would be awesome if someone took it up and got the PR merge-able! https://github.com/invertase/react-native-firebase/pull/3546

**Reply by aloukissas** [reply]
Amazing! I might be able to pick it up but not before Q1 2022.

**Reply by MorenoMdz** [reply]
Hello there, any updates on this plan?

**Reply by mikehardy** [reply]
There are no secret updates, we do our development in the open - there are no updates at the moment but we'd welcome PRs helping to implement it
